metrics/server.go

```diff
@@ -193,17 +192,27 @@ func getMetrics(w http.ResponseWriter, r *http.Request) {
 	application := r.URL.Query().Get("application")
 	if application == "" {
 		http.Error(w, "no application specified", http.StatusBadRequest)
+		return
+	}
+
+	namespace := r.URL.Query().Get("namespace")
+	if namespace == "" {
+		http.Error(w, "no namespace specified", http.StatusBadRequest)
+		return
 	}
-	log.Logger.Tracef("getMetrics called for application %s", application)
+
+	namespaceApplication := fmt.Sprintf("%s/%s", namespace, application)
+
+	log.Logger.Tracef("getAbnDashboard called for application %s", namespaceApplication)
 
 	// identify the routemap for the application
-	namespace, name := util.SplitApplication(application)
+	namespace, name := util.SplitApplication(namespaceApplication)
```

Comment:
this shouldn't be necessary any more. namespace name are namespace, application from a few lines above.

Reply:
Oop sorry